### PR TITLE
Feature: Navigation and homepage

### LINKED ISSUE #2: Add water logging / input buttons
Initial input buttons should have common water sizes in ml, such as 150, 250 and 350 ml

### LINKED ISSUE #4: Add water progress bar
Progress bar should update based on water logging buttons pressed, set current maximum to 1 litre

### PR DESCRIPTION
**Pull request by isabellachen3988**
Added the navigation bar and basic design of the homepage
Home page contains buttons that describe the basic amount of water per contained (in ml)
Upon click, will fill the progress bar on the right, which has a base amount of 1L

closes #2
closes #4

### CODE REVIEW

**Review by KevinZxy1273 — APPROVED** [review]
Looks good, approved

**Review by bingxiaan — APPROVED** [review]
Approved, looks good

File: app/src/main/java/com/dawinder/btnjc/HydroFishViewModel.kt
Comment (marklhj-lee) [inline app/src/main/java/com/dawinder/btnjc/HydroFishViewModel.kt:1]: Change dawinder package name. 
Use rename

File: app/src/main/res/drawable/fish.png
Comment (marklhj-lee) [inline app/src/main/res/drawable/fish.png:1]: Remove this if we are not using it.

File: screenshots/home.png
Comment (marklhj-lee) [inline screenshots/home.png:1]: Remove these pictures and in the future we will replace it with other own images.

File: screenshots/list.png
Comment (marklhj-lee) [inline screenshots/list.png:1]: Remove these pictures and in the future we will replace it with other own images.

File: screenshots/profile.png
Comment (marklhj-lee) [inline screenshots/profile.png:1]: Remove these pictures and in the future we will replace it with other own images.

File: screenshots/search.png
Comment (marklhj-lee) [inline screenshots/search.png:1]: Remove these pictures and in the future we will replace it with other own images.

**Review by marklhj-lee — COMMENTED** [review]
Anyone free can just clone this and commit on this pull repository.
The changes are simple enough.

**Review by marklhj-lee — APPROVED** [review]

**Pull request merged**